__tests__/app.test.js

```diff
@@ -0,0 +1,39 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/topics", () => {
+  describe("GET /api/topics", () => {
+    test("Status 200: responds with an array of topic objects containing the slug and description", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const topics = body.topics;
+          expect(topics.length).toBe(3);
+          topics.forEach((topic) => {
+            expect(topic).toHaveProperty("description", expect.any(String));
+            expect(topic).toHaveProperty("slug", expect.any(String));
+          });
+        });
+    });
+    test("Status 404: responds with 'Path does not exist' message", () => {
```

Comment:
This test is good, but is it specific to the /api/topics endpoint? Or is it more generally for the whole api? Consider moving it into it's own describe block 🙂